### PR TITLE
rules update

### PR DESCRIPTION
**Pull request by mazen160**
Added new rules for FPs we have seen.

To test:

```shell
./trufflehog-fp-analyzer.py --rules rules.yaml --validate-rules  # CI rules validation
./trufflehog-fp-analyzer.py --rules rules.yaml --trufflehog-json tests/fp-4.json  # Test for the observed pattern

### CODE REVIEW

File: rules.yaml [hunk 1]
```diff
@@ -105,3 +105,31 @@ patterns:
...
+      name: withpersona
+      type: regex
+      value: "(https://withpersona.com/api/)[^ ]+(\\.)(jpg|png|jpeg|gif)"
+      reference: "https://withpersona.com/api/v1/verifications/ver_RSskpfbPfWG9Yzy1mfCozgr4/files/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMlJvQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--53f9f680e1d73922d7fb8ab54e4cd8e196b8c28f/center_photo.jpeg"
+      reference_matcher: "https://withpersona.com/api/v1/verifications/ver_RSskpfbPfWG9Yzy1mfCozgr4/files/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMlJvQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--53f9f680e1d73922d7fb8ab54e4cd8e196b8c28f/center_photo.jpeg"
+    commit_hash: "__"
+  - pattern:
+      name: docker image hash
+      type: regex
+      value: '(image:\ )chimebank/server-gsimulator(.)+{IDENTIFIED_STRING}'
```
Comment (arkadiyt) [inline rules.yaml:118]: server-gsimulator is too specific probably - can this be `chimebank/(alphanumeric name):(identified string)`?

File: rules.yaml [hunk 2]
```diff
@@ -105,3 +105,31 @@ patterns:
...
+      name: github diff
+      type: regex
+      value: "(https://github.com/)(.)+(\\#diff-){IDENTIFIED_STRING}"
+      reference: "https://github.com/1debit/server-core/pull/1927/files#diff-67778367d27b9c5afb856770de0680a6R34"
+      reference_matcher: "67778367d27b9c5afb856770de0680a6R34"
+      commit_hash: "__"
+  - pattern:
+      name: a-z
+      type: regex
+      value: "(ABCDEFGHIJKLMNOPQRSTUVWXYZ|abcdefghijklmnopqrstuvwxyz)"
```
Comment (arkadiyt) [inline rules.yaml:132]: This seems very specific

**Review by arkadiyt — APPROVED** [review]

**Comment by arkadiyt** [comment]
Also can you please add a rule excluding ssh keys - we had a lot of alerts like this one: https://app.circleci.com/pipelines/github/1debit/server-ops/167/workflows/74db8a53-648e-4e43-89a2-1774cd64ee9c/jobs/169

File: rules.yaml (hunk 1 above)
Comment (mazen160) [reply rules.yaml:118]: Yes, you're right, It seems I missed it

File: rules.yaml (hunk 2 above)
Comment (mazen160) [reply rules.yaml:132]: Yes, this particular one has been detecting a-z list as an event. The string that has been reported multiple times is `ABCDEFGHIJKLMNOPQRSTUVWXYZ` - for a list of chars for some string generation

File: rules.yaml (hunk 1 above)
Comment (mazen160) [reply rules.yaml]: Updated, thanks for pointing

File: rules.yaml [hunk 3]
```diff
@@ -115,7 +115,7 @@ patterns:
   - pattern:
       name: docker image hash
       type: regex
-      value: '(image:\ )chimebank/server-gsimulator(.)+{IDENTIFIED_STRING}'
+      value: '(image:\ )chimebank/(.)+{IDENTIFIED_STRING}'
```
Comment (arkadiyt) [inline rules.yaml:118]: Still needs the colon

**Comment by mazen160** [comment]
> Also can you please add a rule excluding ssh keys - we had a lot of alerts like this one: https://app.circleci.com/pipelines/github/1debit/server-ops/167/workflows/74db8a53-648e-4e43-89a2-1774cd64ee9c/jobs/169

Added too. Also added a test case:
./trufflehog-fp-analyzer.py --rules rules.yaml --validate-rules  && ./trufflehog-fp-analyzer.py --rules rules.yaml --trufflehog-json tests/ssh-pub-key.json

**Review by arkadiyt — APPROVED** [review]
lgtm

**Pull request merged**